CycleSMCAndroid/src/main/res/values/strings.xml

```diff
@@ -5,13 +5,13 @@
 	<string name="app_name">Cycle San Mateo County</string>
 	<string name="geographicName">San Mateo County</string>
 	<string name="welcomeMessage">Welcome to Cycle San Mateo County!</string>
-	<string name="postUrl">http://www.cyclephilly.org/post/</string>
+	<string name="postUrl">http://www.cyclesmc.org/post/</string>
```

Comment:
I registered cyclesmc.org, so whenever we're ready, we can set up the backend services and point the domain there.